messages.proto

```diff
@@ -85,6 +132,12 @@ message ExecutionInfo {
     StepInfo currentStep = 3;
     /// Stacktrace of the execution. Valid only if there is an error in execution.
     string stacktrace = 4;
+    /// Holds the project name
+    string projectName = 5;
+    /// Holds the environment which is used for execution.
+    string environment = 6;
+    /// Hold the command line arguments.
+    ExecutionArgs args = 9;
```

Comment:
We can add additional information like the number of parallel streams that are running. 
Please try to find such derived values from Gauge that may be useful to the runners.